src/Digdir.Domain.Dialogporten.Application/Features/V1/ServiceOwner/Dialogs/Commands/Delete/DeleteDialogCommand.cs

```diff
@@ -41,6 +41,8 @@ public async Task<DeleteDialogResult> Handle(DeleteDialogCommand request, Cancel
         var resourceIds = await _userService.GetCurrentUserResourceIds(cancellationToken);
 
         var dialog = await _db.Dialogs
+            // Load the elements so that we dont have to load them during cascading events update
+            // TODO: Test this
```

Comment:
Sjekka vi dette?

Reply:
Testa nå. Aggregat logikken vi lagde fokuserer mest på cascading updates fra barn til foreldre. Logikken forstår ikke så godt delete. Det er et par issues som jeg finner i farten:
1. Soft delete modifiserer foreldrene i aggregatet, men holder barna uforandret. Aggregat logikken vår er blind for det som skjer ovenfra og ned. Hadde vi hard deleted dem så hadde EF markert innlastede barn som slettet, som igjen hadde gjort at aggregat logikken hadde sett de.
2. Aggregat logikken vår fungerer nedenfra og opp - dvs dersom vi laster et element uten dens dialog og sletter kun det spesifikke elementet vil logikken vår laste inn dialogen og si i fra at det er modifisert. Men dersom vi sletter en forelder vil ikke forelderen laste inn barn og si i fra at de også er slettet. 

Lagde et [issue](https://github.com/digdir/dialogporten/issues/288) på det. 

Tar det ikke i denne PRen.